.github/workflows/docs.yml

```
@@ -49,7 +49,7 @@ jobs:
           enablement: true
 
       - name: Install documentation dependencies
-        run: pip install -r requirements-docs.txt
+        run: pip install --only-binary :all: -r requirements-docs.txt
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)